packages/shared/src/icons/UniversalIcon.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import * as React from 'react';
+// eslint-disable-next-line no-restricted-imports
+import { Text as RNText } from 'react-native';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+import StyleSheet from '../PlatformStyleSheet';
+import iconsMap from './icons.json';
+
+const sanitizeIconCharacter = (iconCharacter: string) => {
+  if (/^E(?:.{3})$/.test(iconCharacter)) {
+    return String.fromCharCode(parseInt(iconCharacter, 16));
+  }
+  return iconCharacter;
+};
+
+export default function Icon({ name, color = '#46515e', size = 'medium', style }: $FlowFixMe) {
```

Comment:
Consider changing the default color's value to match one of the design token colors?